### PR TITLE
implement create furniture componenet

### PR DESCRIPTION
**Pull request by mingxin-yu**
Finish create furniture components.

### CODE REVIEW

File: src/app/furniture-create/furniture-create.component.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+})
+
+/**
+ * Display a view of create furniture feature
+ * Take a name of a furniture and add to the furniture list
+ */
+export class FurnitureCreateComponent implements OnInit {
+  constructor(private furnitureService: FurnitureService,
+              private roomService: RoomService) {}
+  sampleFurniture = 'Chair';
```
Comment (ghost) [inline src/app/furniture-create/furniture-create.component.ts]: don't assume input

File: src/app/furniture-create/furniture-create.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  sampleRoomId = 0;
```
Comment (ghost) [inline src/app/furniture-create/furniture-create.component.ts]: dont name as sample..

File: src/app/furniture-create/furniture-create.component.ts (hunk 1 above)
Comment (ghost) [reply src/app/furniture-create/furniture-create.component.ts]: name

File: src/app/furniture-create/furniture-create.component.ts (hunk 2 above)
Comment (ghost) [reply src/app/furniture-create/furniture-create.component.ts]: roomId = 0, add comments

File: src/app/furniture-create/furniture-create.component.ts (hunk 1 above)
Comment (mingxin-yu) [reply src/app/furniture-create/furniture-create.component.ts]: Fixed.

File: src/app/furniture-create/furniture-create.component.ts (hunk 2 above)
Comment (mingxin-yu) [reply src/app/furniture-create/furniture-create.component.ts]: Fixed.

**Review by ghost — APPROVED** [review]

**Pull request merged**